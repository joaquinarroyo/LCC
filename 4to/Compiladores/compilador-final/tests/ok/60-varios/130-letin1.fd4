let x : Nat =
  let y : Nat = print "Hola" 11 in
  23

let z : Nat = print "" x

let r : Nat = 1 + (let _ : Nat = 2 in 3)
let k : Nat = 1 + (let _ : Nat = 2 in 3)

let rr : Nat = print "r" r
let kk : Nat = print "k" r
