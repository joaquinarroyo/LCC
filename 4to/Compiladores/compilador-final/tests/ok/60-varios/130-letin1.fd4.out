Hola11
23
r4
k4
